app/controllers/followships_controller.rb

```diff
@@ -5,8 +5,8 @@ def create
       respond_to do |format|
         format.js
       end
-      @notification=Notification.new(user_id:@followship.followee_id,actor_id:@followship.user_id,notification_type:params[:controller],target_type:"follow",target_id:@followship.followee_id)
-      @notification.save
+      Notification.create_notification(user_id:@followship.followee_id,actor_id:@followship.user_id,notification_type:params[:controller],target_type:"follow",target_id:@followship.followee_id)
+
```

Comment:
This code is still not okay. It seems making a method still not give us any advantage. 
Reduce the parameters to method to two.

Reply:
I am not understanding what you're saying, please can you explain it?

Reply:
please can you explain it?

Reply:
What I meant was , you can simply do this.
```
Notification.create_notification( @followship, params)
```
And encapsulate that reset of the logic is your method

Reply:
done